README.md

```
@@ -158,186 +222,47 @@ cd demos/local-fault-injection-demo
 
-- ⭐ **Star this repository** to show your support
+- ⭐ **Star** this repository to show your support
 - 👀 **Watch** for updates on releases and announcements
 - 🔗 **Share** NVSentinel with others who might benefit
 
-## 📄 License
-
-This project is licensed under the Apache License 2.0 - see the [LICENSE](LICENSE) file for details.
+## License
 
+Apache License 2.0. See [LICENSE](LICENSE).
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)